.changeset/shiny-beans-invite.md

```diff
@@ -0,0 +1,5 @@
+---
+'@nordeck/matrix-neoboard-widget': patch
+---
+
+Fix whiteboard initialization when whiteboards loading is slower.
```

Comment:
If this was caused by another issue before doing the release, should we actually mention it,
because the issue doesn't appear in any of the releases?

Reply:
I believe it is better to have a changelog entry in this case, even though there was no release. For transparency reasons and because it is a separate change.